lib/main.js

```diff
@@ -167,7 +184,7 @@ export default {
                 location: {
                   // ansible-lint plus atom-linter combined now lose pathing info for this kind of parsing
                   // prepend dir to file
-                  file: dir + lint_matches[1],
+                  file: dir + path.delimiter + lint_matches[1],
```

Comment:
This does need a character to separate the filename from the path, like another PR to address the same issue my Terraform package had with the same workaround for the atom-linter relative pathing bug also introduced. According to [documentation](https://nodejs.org/api/path.html#path_path_delimiter), this is not the necessary fix though. Ideally atom-linter would fix its relative pathing bug instead, but that does not seem to be a possibility.

Reply:
Maybe it works because it doesn't need the `dir` at all - so only `lint_matches[1]` is required?

Reply:
For the same workaround in the Terraform package, a `/` was concatenated between the `dir` and `file`. I checked and apparently `path` interprets a `/` correctly independent of platform; as weird as that sounds.